### PR TITLE
WIP : Fix bbox webserver

### LINKED ISSUE #44: bounding box is wrong on web server
When running the webrtc you can see the text bounding box seems off, maybe its something to with some of the code not being run in this path 

Screenshot 2024-05-24 at 2 18 09 PM

### PR DESCRIPTION
**Pull request by ShaneKoNaung**
debug_bbox flag is added to print bbox annotations for debugging purpose.

```
python .\ss.py -is  --trans jp,en --debug_bbox --text_detector east
```

bbox_ss

### CODE REVIEW

**Comment by ShaneKoNaung** [comment]
Added `debug_bbox` config to webserv.py
```
python .\webserv.py
```

webserv_bbox

**Comment by ShaneKoNaung** [comment]
Debug_bbox is added to the server.stream_http_video as a config.

```
python -m server.stream_http_video
```
webrtc_bbox

webrtc_bbox2

File: error_analysis_gpto.ipynb
Comment (mattkanwisher) [inline error_analysis_gpto.ipynb:1]: lets not put more files in root, maybe move this to a folder for these notebooks

Comment (ShaneKoNaung) [reply error_analysis_gpto.ipynb:1]: ok, got it.

**Pull request merged**